src/main/java/com/onarandombox/MultiverseCore/utils/permission/PrefixPermission.java

```diff
@@ -0,0 +1,102 @@
+package com.onarandombox.MultiverseCore.utils.permission;
+
+import org.bukkit.Bukkit;
+import org.bukkit.permissions.Permission;
+import org.bukkit.permissions.PermissionDefault;
+import org.bukkit.plugin.PluginManager;
+
+public class PrefixPermission {
```

Comment:
What is PrefixPermission and why is it also registering permissions? Shouldn't the PermissionRegistrar be doing that?

Reply:
PrefixPermission is so that you can dynamically get like `mv.bypass.gamemode.` then you do `PrefixPermission.getPermissionString(<worldname>)` to get `mv.bypass.gamemode.<worldname>`, which is better than using string concatenation